src/context/online.rs

```diff
@@ -55,6 +55,17 @@ pub struct AudioContextOptions {
     pub sample_rate: Option<u32>,
 }
 
+/// Struct returned by `get_output_timestamp` that associate estimated playback
+/// context time (i.e. `current_time - output_latency`) to a monotonic global clock
+/// (i.e. <https://doc.rust-lang.org/std/time/struct.Instant.html>)
+#[derive(Clone, Debug, Default)]
+pub struct AudioTimestamp {
+    /// Represents a point in the time coordinate system of BaseAudioContext’s currentTime.
+    pub context_time: f64,
+    /// Represents a point in the time coordinate system of a Performance interface implementation
+    pub performance_time: f64,
```

Comment:
I strictly followed the spec here, exposing the time in millisecond, but I'm actually not sure this is the best idea.

As I understand it these values are interesting to tightly synchronize other processes (e.g. graphic) to the audio, in such case maybe it would be more handy and useful to directly expose the `std::time::Duration` since `TIME_ORIGIN`, any idea?

Reply:
Maybe we could add some non-spec stuff such as:
- `AudioContext::set_time_origin(i: std::time::Instant)` to allow users to define their own time origin, 
- `AudioTimestamp::performance_duration: std::time::Duration` raw field computed from the default or given time origin